marlowe-dashboard-client/src/Contract/View.purs

```diff
@@ -236,15 +236,15 @@ renderContractCard stepNumber state currentTab cardBody =
       --       so the perceived margins are bigger than we'd want to. To solve this we add negative margin of 4
       --       to the "not selected" cards, a positive margin of 2 to the selected one
       -- Base classes
-      [ "rounded", "overflow-hidden", "flex-shrink-0", "w-contract-card", "h-contract-card", "transform", "transition-transform", "duration-100", "ease-out" ]
+      [ "grid", "grid-rows-contract-step-card", "rounded", "overflow-hidden", "flex-shrink-0", "w-contract-card", "h-contract-card", "transform", "transition-transform", "duration-100", "ease-out" ]
         <> toggleWhen (state ^. _selectedStep /= stepNumber)
             -- Not selected card modifiers
             [ "shadow", "scale-77", "-mx-4" ]
             -- Selected card modifiers
             [ "shadow-lg", "mx-2" ]
   in
     div [ classNames contractCardCss ]
-      [ div [ classNames [ "flex", "overflow-hidden" ] ]
+      [ div [ classNames [ "flex" ] ]
```

Comment:
the `overflow-hidden` doesn't play well with the grid layout (it prevents this element from taking up any space), but I realised that it isn't necessary anyway - the `overflow-hidden` in the parent element takes care of this